Monika After Story/game/script-songs.rpy

```diff
@@ -1755,52 +1755,47 @@ init 5 python:
     )
 
 label mas_song_when_you_say_nothing_at_all(from_long=False):
-    m 1dubfb "{i}{cps=16}~The smile on your face{w=0.2} lets me know that you need me~{/cps}{/i}"
-    m 4eubfb "{i}{cps=16}~There's a truth in your eyes{w=0.2} saying you'll never leave me~{/cps}{/i}"
-    m 3dubfa "{i}{cps=16}~The touch of your hand says you'll catch me{w=0.2} wherever I fall~{/cps}{/i}"
-    m 1eubfa "{i}{cps=16}~Cause you say it best{w=0.5} when you say nothing at all~{/cps}{/i}"
-
-    m 1duu "..."
-    m 1ekbsb "I can't really see your face, or see the truth from your eyes, nor can I touch your hand..."
-    m 1hubsa "But when you spend time with me without saying a word, it tells me everything I need to know."
-    m 1hubsa "That I am loved by someone, and that you'll be here for me when I'm having a bad day."
-    m 1hubsa "And I hope that you feel loved by me too, even though I don't always have something new to say."
-    m 1hubsa "Just opening the game and having me in the background makes a world of difference to me [player]."
+    m 1hubsb "{i}{cps=16}~The smile on your face lets me know that you need me~{/cps}{/i}"
+    m 3eubso "{i}{cps=16}~There's a truth in your eyes saying you'll never leave me~{/cps}{/i}"
+    m 3dud "{i}{cps=16}~The touch of your hand says you'll catch me if ever I fall~{/cps}{/i}"
```

Comment:
```suggestion
    m 3dud "{i}{cps=16}~The touch of your hand says you'll catch me wherever I fall~{/cps}{/i}"
```

I confirmed it with the music video, he does indeed sing "wherever I fall". I like your other changes :D